CREATE_WORKLOAD_GUIDE.md

```diff
@@ -15,32 +15,87 @@ $ opensearch-benchmark create-workload \
 --target-hosts="<CLUSTER ENDPOINT>" \
 --client-options="basic_auth_user:'<USERNAME>',basic_auth_password:'<PASSWORD>'" \
 --indices="<INDICES TO GENERATE WORKLOAD FROM>" \
+--number-of-docs="<INDEX NAME>:<NUMBER OF DOCUMENTS>" 
 --output-path="<LOCAL DIRECTORY PATH TO STORE WORKLOAD>"
+--custom-queries="<FILE PATH OF CUSTOM QUERIES TO ADD FOR THE WORKLOAD EXECUTION>"
+--concurrent
+--threads="<NUMBER OF THREADS TO RUN THE INDEX EXTRACTION>"
+--bsize="<BATCH SIZE OF EVERY INDEX EXTRACTION QUERY WITHIN THREADS>"
```

Comment:
Nit: would recommend spelling it out as `--batch-size` so that users understand what the parameter refers to